examples/aws/api_gateway_with_domain/api_gateway_with_domain.tf

```
@@ -27,7 +27,8 @@ module "lambda_api_gateway_with_domain" {
   domain = {
     certificate_arn = data.aws_acm_certificate.acm_certificate.arn
     zone_name       = "they-code.de."
-    domain          = "they-test-lambda.they-code.de"
+    # while developing the mtls implementation, domain below without timestamp was used, which now causes host of lambda to only resolve after waiting substantial amount of time (approx. 1hr)
```

QoL:

I don't think the comment fits the line anymore. Perhaps rather something like:

```suggestion
    # including a timestamp into the domain to avoid issues where we can't redeploy because it takes around 1hour for AWS to not throw an error on redeployment. Somewhat similar to s3 buckets which take a day after destruction to be able to recreate it.
```

The suggestion from me might also not the best, perhaps you find a better one or we just keep it as is.

---

came up with sth shorter :D!!!
`# used domain without timestamp during mtls development, which now delays lambda host resolution (approx. 1hr).`

---

thx for making me rethink this, I hated the long comment while writing it